src/common.ts

```diff
@@ -54,4 +55,58 @@ export class Common {
     logger.debug(`loginUrl: ${loginUrl}`);
     return loginUrl;
   }
+
+  // fields property is passed in because the consumers of this method have performed the decrypt.
+  // This is so we don't have to call authInfo.getFields(true) and decrypt again OR accidentally save an
```

Comment:
The comment is incomplete

Reply:
This was added i this PR, I merged @mshanemc changes before they were available in main.

I am happy to fix this Shane, just give me the complete thought.